### PR TITLE
 feat: multiple issue templates, PR template, initial structure.

### PR DESCRIPTION
**Pull request by Lavanyagaur22**
Referring to #8  
Multiple issue templates, pr template, Contributing template (to be updated) and initial project structure added. Please have a look at it.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,23 @@
+*.iml
+build
+.gradle
+/local.properties
+.idea
+/.idea/caches/build_file_checksums.ser
```
Comment (wtrocki) [inline .gitignore]: Do we need those?

**Review by wtrocki — COMMENTED** [review]
Do you think is ok to have app in example folder? Having app in root is antipatern for library projects. We want root folder as clean as possible

**Comment by wtrocki** [comment]
See my comment https://github.com/aerogear/graphql-android/issues/8#issuecomment-497848662

No strong opinion here but it will be kinda messy to have app in the root

**Comment by Lavanyagaur22** [comment]
> See my comment [#8 (comment)](https://github.com/aerogear/graphql-android/issues/8#issuecomment-497848662)
> 
> No strong opinion here but it will be kinda messy to have app in the root

@wtrocki Actually I followed the pattern which the standard open source projects like https://github.com/square/picasso etc are following, they have it in root so, what do you say?

File: .gitignore (hunk 1 above)
Comment (harshithdwivedi) [reply .gitignore]: +1,
I'd recommend gitignoring the entire .idea directory

File: app/build.gradle [hunk 2]
```diff
@@ -0,0 +1,33 @@
+apply plugin: 'com.android.application'
+
+apply plugin: 'kotlin-android'
+
+apply plugin: 'kotlin-android-extensions'
+
+android {
+    compileSdkVersion 28
+    defaultConfig {
+        applicationId "com.lavanya.graphqlandroid"
```
Comment (harshithdwivedi) [inline app/build.gradle]: Maybe change the package name to ```org.aerogear.graphql.androidsample```?

@wtrocki what do you say?

**Review by harshithdwivedi — COMMENTED** [review]
Looks good, just a minor change.

**Comment by wtrocki** [comment]
Both work for me. Picasso is also using picasso-sample folder for this. We use app but that is ok too as long we document it

**Comment by wtrocki** [comment]
Good to go from my side

File: app/build.gradle (hunk 2 above)
Comment (danielpassos) [reply app/build.gradle]: I think Android is not needed in the package `org.aerogear.graphql.sample` is good enough

**Comment by danielpassos** [comment]
`app` folder in the root folder is not a problem, but usually when it's not a real app (I mean app only on the repo) we usually rename it for something like example

**Comment by Lavanyagaur22** [comment]
@harshithdwivedi Please have a look at it.

**Comment by Lavanyagaur22** [comment]
> `app` folder in the root folder is not a problem, but usually when it's not a real app (I mean app only on the repo) we usually rename it for something like example

@danielpassos Yeah renamed it to sample app :)

**Review by wtrocki — APPROVED** [review]

**Review by danielpassos — CHANGES_REQUESTED** [review]
Gradle folder should not be ignored (see .gitignore)

If you try to run `./gradlew clean` it will cause an error because the project does not contain the Gradle Wrapper

PS: You can stole some ideas from the [aerogear-android-sdk/.gitignore](https://github.com/aerogear/aerogear-android-sdk/blob/master/.gitignore)

**Comment by danielpassos** [comment]
@Lavanyagaur22 I'm not a big fan of using _ in a folder. I really prefer only a sample or example but I don't wanna block you because of this. It's only my personal preference, not a bad thing

**Comment by harshithdwivedi** [comment]
+1, please don't gitignore gradle folder 
You can gitignore .gradle though.

Here's a good ref (it has some .idea files which we don't need so ignoring the entire .idea folder is a good idea)

https://github.com/github/gitignore/blob/master/Android.gitignore

**Comment by Lavanyagaur22** [comment]
@harshithdwivedi @danielpassos Changed the name to sample and modified .gitignore

**Review by danielpassos — APPROVED** [review]

**Comment by danielpassos** [comment]
approval

**Pull request merged**